.github/workflows/create-release.yaml

```diff
@@ -262,12 +316,24 @@ jobs:
           git tag ${{ github.event.inputs.name }}
           git push origin ${{ github.event.inputs.name }}
 
+      - name: Check if any PR may have been merged before creating the tag and draft release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          COMMIT_SHA=$(gh api repos/${{ env.KYMA_BTP_MANAGER_REPO }}/commits -q '.[0].sha')
+          echo "Latest commit sha: $COMMIT_SHA"
+          echo "Bump sha: ${{ needs.bump-sec-scanners-config.outputs.latest_commit}}"
+          if [ "$COMMIT_SHA" != "${{ needs.bump-sec-scanners-config.outputs.latest_commit}}" ]; then
+            echo "::error ::a PR may have been merged before the tag and draft release were created. Delete the tag and the draft release. Don't rerun this workflow. Create a new release with the same version"
+            exit 1 
+          fi
+
     outputs:
       release_id: ${{ steps.create-draft.outputs.release_id }}
 
   publish-release:
     name: Publish release
-    needs: [create-draft, run-unit-tests, run-e2e-tests, run-e2e-upgrade-tests, run-stress-tests, run-e2e-upgrade-while-deleting-tests]
+    needs: [create-draft, bump-sec-scanners-config]
```

Comment:
This `bump-sec-scanners-config` unlike in KEB is not needed here because we don't checkout here to the commit created in the bump step.

Reply:
In KEB we create a helm package and this is why we need to checkout to the latest version. Here we run only the publish where only the draft ID is needed.